### DISCUSSION TITLE
Problem Connecting to Metamask

### DISCUSSION THREAD

**Discussion by skeeterrabbit** [smartcontractkit/full-blockchain-solidity-course-py#1466] [Q&A] [answered]
Hi all,
I'm at the end of Lesson 1: Welcome to Remix! Simple Storage from https://youtu.be/M576WGiDBdQ?t=7513
My problem is that my brave browser is connecting to my brave wallet instead of the metamask wallet that I had created.  Since I have 2 wallets on my brave wallet.  How do I change to the metamask wallet instead of the brave wallet?

Thanks in advance.

**Comment by cromewar** [comment, marked as answer]
Hello @skeeterrabbit this is due to the configuration of the browser and how it takes the default wallet, as braves comes with a integrated one is going to choose that one instead of metamask, the easiest way to solve this is just to use another browser.

Cheers.
